about/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <h1>Intellect Fox</h1>
+        <h3>Front-end 4</h3>
+        <hr/>
+        <h5>Почему я выбрала направление Front-end разработки &#10067</h5>
+    </head>
+    <body>
+        <p>Я очень долго порывалась начать обучение в сфере IT (<del>а хочу я туда конечно же ради денег &#128176</del>), но не знала, с какой сторона подступиться, так как вообще не имею опыта в программировании.
+        <br/>На сайте курса говорится:
+            <blockquote cite="https://intellectfox.by/">
+                <strong>Для того, чтобы <em>«войти»</em> в IT как инженер, профессия Front-end разработчика — отличный вариант для тех,
+                    <br/>кто никогда не занимался программированием. Технологии, которые необходимы для этой профессии, 
+                    <br>достаточно легко освоить и язык JavaScript — один из самых простых в программировании.</strong>
+            </blockquote>
+            Из чего я поняла, что данное направление - отличное начало пути. &#9786
+        </p>
```

Comment:
It's bad practice to separate `p` tag with help `br`.  If you want to get some text by the new line, create another tag `p`.

Reply:
created anther tags <p>.